core/trino-main/src/main/java/io/trino/server/security/oauth2/IdpClient.java

```diff
@@ -19,7 +19,7 @@
 
 import static java.util.Objects.requireNonNull;
 
-public interface OAuth2Client
+public interface IdpClient
```

Comment:
What's the rationale for renaming this? If we want to be clearer as to eliminate confusion as to what OAuth2 we're talking to perhaps `OAuth2IdpClient`?

Reply:
Reverted